README.md

```diff
@@ -58,12 +58,15 @@ class App extends Component {
 
 const stories = [
  	//🆕! Rendering Components instead of video or images can now be done by passing 
-	//a content property into the story.
+	//a content property into the story. There is also an 2 props, 'action' (fn) and 'isPaused'(bool). 
```

Comment:
Nice. Can you change it to `The props contain properties 'action'(fn) and 'isPaused'(bool)`?